src/ray/common/ray_config_def.h

```diff
@@ -183,7 +183,7 @@ RAY_CONFIG(int, object_manager_timer_freq_ms, 100)
 
 /// Timeout, in milliseconds, to wait before retrying a failed pull in the
 /// ObjectManager.
-RAY_CONFIG(int, object_manager_pull_timeout_ms, 10000)
+RAY_CONFIG(int, object_manager_pull_timeout_ms, 1000)
```

Comment:
Can you keep the default to 10k to avoid regressions?

Reply:
Basically I think the concern is that 10k is already causing spurious duplicate pulls, 1k can make it much worse. In the future, we should have better logic about cancelling pulls when we issue a new request.

Reply:
Sure, but this PR will drastically decrease the amount of retries then.